arangod/Aql/DocumentProducingHelper.h

```diff
@@ -138,21 +139,21 @@ struct DocumentProducingFunctionContext {
   transaction::Methods& _trx;
   PhysicalCollection& _physical;
   Expression* _filter;
-  arangodb::aql::Projections const& _projections;
-  arangodb::aql::Projections const& _filterProjections;
+  aql::Projections const& _projections;
+  aql::Projections const& _filterProjections;
   uint64_t _numScanned;
   uint64_t _numFiltered;
 
   std::unique_ptr<DocumentProducingExpressionContext> _expressionContext;
 
   /// @brief Builder that is reused to generate projection results
-  arangodb::velocypack::Builder _objectBuilder;
+  velocypack::Builder _objectBuilder;
 
   /// @brief set of already returned documents. Used to make the result distinct
   containers::FlatHashSet<LocalDocumentId> _alreadyReturned;
 
-  RegisterId const _outputRegister;
   Variable const* _outputVariable;
+  RegisterId const _outputRegister;
```

Comment:
It is an unrelated reordering of members to make the type more compact.

Reply:
Hmm is RegisterId has other alignment?

Reply:
`RegisterId` has the size of a uint32_t.
So having a `RegisterId` and then a pointer and then a bool would waste space.
Rearranging them so that we have a pointer, then a uint32_t-sized type and then a boolean is likely better.